utility/src/main/java/org/oppia/util/parser/HtmlParsingModule.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.util.parser
+import android.app.Application
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import javax.inject.Named
+import javax.inject.Singleton
+
+/** Provides Html parsing dependencies. */
+@Module
+class HtmlParsingModule {
+  @Provides
+  @Singleton
+  @ApplicationContext
+  fun provideApplicationContext(application: Application): Context {
+    return application
+  }
+
+  // TODO(#59): Remove this provider once all modules have access to the @ApplicationContext qualifier.
```

Comment:
Similarly, you should be able to just inject Context directly into HtmlParser. Please remove this module since it shouldn't be necessary.

Reply:
removed.